roxctl/central/export/export.go

```diff
@@ -0,0 +1,25 @@
+package export
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/roxctl/central/export/deployments"
+	"github.com/stackrox/rox/roxctl/central/export/images"
+	"github.com/stackrox/rox/roxctl/central/export/nodes"
+	"github.com/stackrox/rox/roxctl/central/export/pods"
+	"github.com/stackrox/rox/roxctl/common/environment"
+)
+
+// Command defines the central command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "export",
+		Short: "Commands related to exporting data from Central.",
+	}
+	c.AddCommand(
+		deployments.Command(cliEnvironment),
+		images.Command(cliEnvironment),
+		nodes.Command(cliEnvironment),
+		pods.Command(cliEnvironment),
+	)
+	return c
```

Comment:
As an alternative way to have the flag defined on the parent command with the 10 minute default compared to each command itself, you can add it here as a persistent flag and all sub commands will have access to it (thus not having to re-define the flag for each command).

Reply:
Done